lib/include/jxl/encode.h

```diff
@@ -251,6 +251,24 @@ typedef enum {
    */
   JXL_ENC_OPTION_MODULAR_PREDICTOR = 27,
 
+  /** Fraction of pixels used to learn MA trees as a percentage. -1 = default,
+   * 0 = no MA and fast decode, 50 = the default, 100 = All.
+   */
+  JXL_ENC_OPTION_MODULAR_TREE_PERCENT = 28,
```

Comment:
These enum names are already long enough, but maybe `JXL_ENC_OPTION_MODULAR_MA_TREE_LEARNING_PERCENT` is slightly more informative.

Reply:
nothing against the longer name in this case, done